packages/dapp/.env.example

```diff
@@ -1,4 +1,5 @@
 # infura key created at infura.io
 INFURA_ID=XXXXXX
 NEXT_PUBLIC_JSON_RPC_URL=XXXXXX
-NEXT_PUBLIC_WALLET_CONNECT_ID=XXXXXX
\ No newline at end of file
+NEXT_PUBLIC_WALLET_CONNECT_ID=XXXXXX
```

Comment:
Make sure to pass secrets walletconnectID

Reply:
This should be explained in the README.md I think.